addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -0,0 +1,186 @@
+odoo.define('knowledge.dialogs', function (require) {
+'use strict';
+
+const Dialog = require('web.Dialog');
+const { _t } = require('web.core');
+
+const MoveArticleToDialog = Dialog.extend({
+    template: 'knowledge.knowledge_move_article_to_modal',
+    /**
+     * @override
+     * @param {Widget} parent
+     * @param {Object} options
+     * @param {Object} props
+     */
+    init: function (parent, options, props) {
+        // Set default options:
+        options.title = options.title || _t('Move an Article');
+        options.buttons = options.buttons || [{
+            text: _t('Ok'),
+            classes: 'btn-primary',
+            click: this.save.bind(this)
+        }, {
+            text: _t('Cancel'),
+            close: true
+        }];
+        this._super(...arguments);
+        this.props = props;
+        // Set template variables:
+        const { state } = props;
+        this.article_name = state.data.name;
+    },
+
+    /**
+     * @override
+     * @returns
+     */
+    start: async function () {
+        const result = await this._super(...arguments);
+        this.initSelect2();
+        return result;
+    },
+
+    /**
+     * @returns {JQuery}
+     */
+    getInput: function () {
```

Comment:
Seems a bit unnecessary to extract a small one-liner into a method ^^

(Side note: `this.$el.find('input');` -> `this.$('input');`)